modules/cluster-logging-updating-logging-to-current.adoc

```diff
@@ -0,0 +1,218 @@
+:_content-type: PROCEDURE
```

Comment:
This file is missing the required module file metadata `// Module file include in the following assemblies:`. 

See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata